src/screencast/wlr_screencast.c

```diff
@@ -118,6 +118,23 @@ static void wlr_frame_buffer_chparam(struct xdpw_screencast_instance *cast,
 	wlr_frame_buffer_destroy(cast);
 }
 
+static void wlr_frame_linux_dmabuf(void *data,
+		struct zwlr_screencopy_frame_v1 *frame,
+		uint32_t format, uint32_t width, uint32_t height) {
+	//struct xdpw_screencast_instance *cast = data;
```

Comment:
No need to leave this comment.

Reply:
Ok, removed that. Will add that line again, when i actual use this function. Had commented that out, because of that unused variable compiler warning.

Reply:
Thanks, that's better. I think at one point I had even written a noop function just for an unused event to avoid the warning too.